<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SignUp</class>
 <widget class="QDialog" name="SignUp">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>854</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>LibManager</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:white</string>
  </property>
  <widget class="QLabel" name="SignUp_text">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>30</y>
     <width>171</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Calibri</family>
     <pointsize>22</pointsize>
     <italic>false</italic>
     <fontweight>Light</fontweight>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:black;
font: 300 22pt &quot;Calibri&quot;;
background-color: transparent;</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:26pt;&quot;&gt;Sign Up&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="Username_text">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>235</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);
background-color: transparent;</string>
   </property>
   <property name="text">
    <string>Username:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="UsernameLineEdit">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>260</y>
     <width>341</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Calibri</family>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
	background-color: rgb(214, 214, 214);
	border-radius: 15px;
	color: black; 
    font-size: 16px;
    padding-left: 6px; 
}

QLineEdit::placeholder {
	color: rgb(141,141,141); 
    font-style: italic; 
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>Choose a username</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Back_Button">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>20</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::FocusPolicy::StrongFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	border-radius:15px;
	color: black;
	
	font: 300 10pt &quot;Calibri&quot;;
		
	background-color: qlineargradient(spread:reflect, x1:0.484, y1:0.534091, x2:0.479, y2:1, stop:0 rgba(90, 191, 47, 255), stop:1 rgba(13, 134, 0, 255));
}
QPushButton:hover {
	background-color: rgb(136, 202, 129);
}</string>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <widget class="QLabel" name="Password_text">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>316</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);
background-color: transparent;</string>
   </property>
   <property name="text">
    <string>Password:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Password_Line_Edit">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>340</y>
     <width>341</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Calibri</family>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
	background-color: rgb(214, 214, 214);
	border-radius: 15px;
	color: black; 
    font-size: 13px;
    padding-left: 6px; 
}

QLineEdit::placeholder {
	color: rgb(141,141,141); 
    font-style: italic; 
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::EchoMode::Password</enum>
   </property>
   <property name="placeholderText">
    <string>Please set a password of at least 8 characters.</string>
   </property>
  </widget>
  <widget class="QLabel" name="Mail_text">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>155</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);
background-color: transparent;</string>
   </property>
   <property name="text">
    <string>E-mail:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Mail_Line_Edit">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>180</y>
     <width>341</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Calibri</family>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
	background-color: rgb(214, 214, 214);
	border-radius: 15px;
	color: black; 
    font-size: 16px;
    padding-left: 6px; 
}

QLineEdit::placeholder {
	color: rgb(141,141,141); 
    font-style: italic; 
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>Please enter your e-mail</string>
   </property>
  </widget>
  <widget class="QLabel" name="SignUpScreen_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>427</width>
     <height>480</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(255,255,255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="AuthScreen1_2">
   <property name="geometry">
    <rect>
     <x>427</x>
     <y>0</y>
     <width>428</width>
     <height>480</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image:url(C:/LibManager/LibManager/LBResources/authScreen_imageV2.png)</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="LibManager_logo">
   <property name="geometry">
    <rect>
     <x>465</x>
     <y>70</y>
     <width>349</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(C:/LibManager/LibManager/LBResources/LibManager_logo_revised.png);
background-color:transparent;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="signupButton">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>400</y>
     <width>341</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	border-radius:15px;
	color: black;
	
	font: 300 10pt &quot;Calibri&quot;;
		
	background-color: qlineargradient(spread:reflect, x1:0.484, y1:0.534091, x2:0.479, y2:1, stop:0 rgba(90, 191, 47, 255), stop:1 rgba(13, 134, 0, 255));
}
QPushButton:hover {
	background-color: rgb(136, 202, 129);
}</string>
   </property>
   <property name="text">
    <string>Sign up</string>
   </property>
  </widget>
  <widget class="QPushButton" name="user_Button">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>100</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::FocusPolicy::StrongFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	border-radius:15px;
	color: black;
	
	font: 300 10pt &quot;Calibri&quot;;
		
	background-color: qlineargradient(spread:reflect, x1:0.484, y1:0.534091, x2:0.479, y2:1, stop:0 rgba(90, 191, 47, 255), stop:1 rgba(13, 134, 0, 255));
}

QPushButton:hover {
	background-color: rgb(136, 202, 129);
}</string>
   </property>
   <property name="text">
    <string>User</string>
   </property>
  </widget>
  <widget class="QPushButton" name="admin_Button">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>100</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::FocusPolicy::StrongFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	border-radius:15px;
	color: black;
	
	font: 300 10pt &quot;Calibri&quot;;
		
	background-color: qlineargradient(spread:reflect, x1:0.484, y1:0.534091, x2:0.479, y2:1, stop:0 rgba(90, 191, 47, 255), stop:1 rgba(13, 134, 0, 255));
}
QPushButton:hover {
	background-color: rgb(136, 202, 129);
}</string>
   </property>
   <property name="text">
    <string>Admin</string>
   </property>
  </widget>
  <widget class="QLabel" name="welcome_text">
   <property name="geometry">
    <rect>
     <x>484</x>
     <y>170</y>
     <width>312</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 18pt &quot;Calibri&quot;;
color: white;
background-color:transparent;</string>
   </property>
   <property name="text">
    <string>Welcome to LibManager!</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>240</y>
     <width>361</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:transparent;
color: white;</string>
   </property>
   <property name="text">
    <string>Organize, track, and optimize your library effortlessly.</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>497</x>
     <y>280</y>
     <width>291</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:transparent;
color: white;</string>
   </property>
   <property name="text">
    <string>Create, edit, and maintain records of books.</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>525</x>
     <y>320</y>
     <width>231</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:transparent;
color: white;</string>
   </property>
   <property name="text">
    <string>New to LibManager? Sign up now.</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
  </widget>
  <zorder>AuthScreen1_2</zorder>
  <zorder>SignUpScreen_2</zorder>
  <zorder>SignUp_text</zorder>
  <zorder>Username_text</zorder>
  <zorder>UsernameLineEdit</zorder>
  <zorder>Back_Button</zorder>
  <zorder>Password_text</zorder>
  <zorder>Password_Line_Edit</zorder>
  <zorder>Mail_text</zorder>
  <zorder>Mail_Line_Edit</zorder>
  <zorder>LibManager_logo</zorder>
  <zorder>signupButton</zorder>
  <zorder>user_Button</zorder>
  <zorder>admin_Button</zorder>
  <zorder>welcome_text</zorder>
  <zorder>label</zorder>
  <zorder>label_2</zorder>
  <zorder>label_3</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
